src/utils/drive.utils.ts

```diff
@@ -0,0 +1,10 @@
+import dayjs from 'dayjs';
+
+export const humanFileSize = (size: number) => {
```

Comment:
I bet it makes more sense to call this `format.utils.ts` as this is mostly related to formatting stuff, is not Drive only, we can use this for whatever needs formatting right? Also, why don't we use the same pattern as in the other utils files? 

```

export class FormatUtils {

    function formatBytes(bytes: number) {
            const i = size == 0 ? 0 : Math.floor(Math.log(size) / Math.log(1024));
            return Number((size / Math.pow(1024, i)).toFixed(2)) * 1 + ' ' + ['B', 'kB', 'MB', 'GB', 'TB'][i];
    }
    
    function formatDate(date: string | Date) {
        return dayjs(date).format('D MMMM, YYYY [at] HH:mm');
    };
}